public/app/features/inspector/InspectDataOptions.tsx

```diff
@@ -40,20 +40,19 @@ export const InspectDataOptions = ({
   const styles = useStyles2(getPanelInspectorStyles2);
 
   const panelTransformations = panel?.getTransformations();
-  const showPanelTransformationsOption =
-    Boolean(panelTransformations?.length) && (transformId as any) !== 'join by time';
+  const showPanelTransformationsOption = Boolean(panelTransformations?.length);
```

Comment:
Didn't you loose one part of the condition here?

Reply:
@mckn ah yes thanks for spotting this - i did remove part of the condition, but that's because (according to the types...) it's not possible to have a `transformId` of `join by time`.

i can't quite figure out if it's a transformation that we used to have and has been removed (in which case we don't need the condition anymore), or if it's a transformation that's been renamed (in which case we do, but with a different name) 😬 any thoughts?

Reply:
@mckn have double checked this and it looks like `join by time` is now a subset of the `joinByField` transformation. given all this condition does is controls whether to show the `Apply panel transformations` control in the `inspect data` tab:
![image](https://github.com/grafana/grafana/assets/20999846/eb0d545c-738a-43f5-80b2-2c1696e2f28b)

i think it's safe to remove it and makes the logic simpler. now we're saying we'll show the control if you have **any** transformations. maybe in the case of `joinByField(time)` toggling that control will make no difference to the data in the table below, but i think that's ok? 

lmk if you disagree and i can revert/look into it more